.travis.yml

```diff
@@ -3,27 +3,38 @@ language: julia
 os:
   - linux
   - osx
+  - windows
 julia:
-  - 1.0
-  - 1.1
-  - 1.2
-  - 1.3
-  - 1.4
+  - 1.0   # LTS
+  - 1.3   # Release Invenia uses
```

Comment:
Do we want to add:
```suggestion
  - 1  # stable
```

Reply:
Yes, but probably in a separate PR because I don't think the specimen files generated for 1.3 work properly on 1.5.

Reply:
That's part of what #80 is about